docs/api/v1.24.md

```diff
@@ -1659,6 +1659,7 @@ Build an image from a Dockerfile
 **Example request**:
 
     POST /v1.24/build HTTP/1.1
+    Content-Type: application/tar
```

Comment:
I went with "application/tar" because that's what I saw the docker-build command uses: https://github.com/docker/docker/blob/master/client/image_build.go#L32

🐈